src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -369,7 +384,7 @@ ManifestCache::load (
             beast::detail::base64_decode(configManifest));
         if (! mo)
         {
-            JLOG (j_.error()) << "Malformed manifest in config";
+            JLOG (j_.error()) << "Malformed token manifest in config";
```

Comment:
Oh!  Then perhaps the message "Malformed validator_token in config" would be a good choice.